.github/actions/build-fixtures/action.yaml

```
@@ -0,0 +1,45 @@
+name: Build and Package Fixture Release
+inputs:
+  release_name:
+    description: "Name of the fixture release"
+    required: true
+  uv_version:
+    description: "Version of UV to install"
+    required: true
+  python_version:
+    description: "Version of Python to install"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Install uv ${{ inputs.uv_version }} and python ${{ inputs.python_version }}
+      uses: astral-sh/setup-uv@0c5e2b8115b80b4c7c5ddf6ffdd634974642d182
+      with:
+        enable-cache: false
+        version: ${{ inputs.uv_version }}
+        python-version: ${{ inputs.python_version }}
+    - name: Install EEST
+      shell: bash
+      run: uv sync --no-progress --extra fill --extra test
```

```suggestion
      run: uv sync --no-progress --extra test
```

Tried out locally just now with #1654 and "fill" extra doesn't work anymore nor is required I think.

---

Thx!